elements/stackhpc-ipa-hardware-managers/package-installs.yaml

```diff
@@ -1 +1,3 @@
 dmidecode:
+pciutils:
+ethtool:
```

Comment:
Should we add a comment to show which plugin requires these packages? And I guess also retrofit one to dmidecode.